src/pages/nonMbLookup/components/InputWrapper.tsx

```diff
@@ -56,64 +57,72 @@ const InputWrapper = ({ btnOn, btnOff, isReadyRequest, dataStatus }: InputProps)
     }
   }, [bookerName, birth, number, password]);
 
+  const postUserData = async (postData: postGuestBookingReq) => {
+    const bookingData = await mutateAsync(postData);
+    console.log(bookingData);
+    if (bookingData === 404) {
+      resetForm();
+      dataStatus(404);
+    } else {
+      dataStatus(200);
+      navigate("/lookup", { state: bookingData });
+    }
+  };
+
```

Comment:
p3) 여기서 데이터 전송이 되는 동안은 실행되지 않게 하는 코드 추가해주세요!
```suggestion
  const postUserData = async (postData: postGuestBookingReq) => {
    if(isPending) return; 
    
    const bookingData = await mutateAsync(postData);
    console.log(bookingData);
    if (bookingData === 404) {
      resetForm();
      dataStatus(404);
    } else {
      dataStatus(200);
      navigate("/lookup", { state: bookingData });
    }
  };
```